front/components/assistant_builder/TeamSharingSection.tsx

```diff
@@ -112,15 +111,16 @@ export function TeamSharingSection({
             private version.
           </div>
           <div>
-            <Tooltip label="Coming soon: you can create a duplicate to tweak your own, private version">
+            <Link
+              href={`/w/${owner.sId}/builder/assistants/${agentConfigurationId}?flow=personal_assistants&copy=true`}
+            >
               <Button
                 variant="secondary"
                 size="sm"
                 label="Create a duplicate"
                 onClick={() => setShowCreateDuplicateDialog(true)}
```

Comment:
this is unexpected as you show a dialog and you are wrapped in a Link at the same time?